code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -1,14 +1,103 @@
+import Radium from 'radium';
 import React from 'react';
 import { STAGE_PROGRESS_TYPE } from './types';
 import { saveAnswersAndNavigate } from '../../levels/saveAnswers';
+import color from '../../color';
+
+var style = {
+  overviewContainer: {
+    display: 'table-cell',
+    verticalAlign: 'middle',
+    paddingRight: '10px'
+  },
+  headerContainer: {
+    padding: '5px 8px',
+    backgroundColor: color.lightest_gray,
+    border: `1px solid ${color.lighter_gray}`,
+    borderRadius: '5px'
+  },
+  dot: {
+    puzzle: {
+      display: 'inline-block',
+      width: '24px',
```

Comment:
would it make sense to use variables for some of these values that are repeated a few times?